enlighten/Graph.py

```diff
@@ -276,7 +276,7 @@ def enable_wavenumbers(self, flag):
 
     ##
     # Only sets the "intention" to use the specified axis label; in reference-
-    # based techniques, don't actually switch to the target axis until processing
+    # based views, don't actually switch to the target axis until processing
```

Comment:
We need to figure out a consistent way to say this.  There might need to be a new "Nomenclature" section in the manual, and README.  In my mind:

- "Mode" is a way the _software_ behaves.  There will be a significant and growing set of on-screen options which vary between these three labeled sets: Raman, Non-Raman, and Expert.  
- Now, the fact that I named a "feature set" to "Raman" doesn't mean that "Raman" _isn't also a spectroscopic technique.  It is._  "Raman" will be offered as a convenience "application mode" (in which it shows all Raman-relevant widgets and features), but it also, scientifically and in our comments / documentation, remains a "Technique".  Basically though, at this time, we don't offer any techniques _other_ than Raman while in "Raman Mode", so "Raman" is being used both as a Mode (in page navigation) and as a technique (in internal comments).
- Raman is a non-reference technique.  It is a spectroscopic technique which does not use a reference.
- All reference-based techniques (Refl, Trans, Abs, Irrad etc) will be offered in the  "Non-Raman mode".  However, there will be some non-Raman techniques which don't use references.  So basically, we can't use either Mode or View as a surrogate to decide whether the current technique needs a reference.  
- We can assume that if we're in Raman mode, then we are in a non-reference technique (because Raman mode only supports Raman technique); however the converse is not true, because for instance "Emission" or "Fluorescence" are non-Raman techniques which don't use a reference.
- I don't remember what point I was trying to make, other than that we'll continue to tweak some of these comments.